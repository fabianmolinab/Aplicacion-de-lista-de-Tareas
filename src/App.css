:root {
	--verde: #80BE4E;
	--azul: #007AEA;
}

.contenedor {
	box-shadow: 0 2px 10px rgba(139,139,139,.25);
	border-radius: 3px;
	display: grid;
	height: 100%;
	grid-template-rows: auto auto 1fr;
	overflow: auto;
	max-width: 700px;
	max-height: 90vh;
	width: 100%;
}