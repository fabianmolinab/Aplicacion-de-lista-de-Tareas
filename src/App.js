import React, { useState } from 'react'
import styled from 'styled-components';

import { Header } from './Componentes/Header';
import { FormularioTareas } from './Componentes/FormularioTareas';


const Contenedor__app = styled.div`
box-shadow: 0 2px 10px rgba(139,139,139,.25);
border-radius: 3px;
display: grid;
height: 100%;
grid-template-rows: auto auto 1fr;
overflow: auto;
max-width: 700px;
max-height: 90vh;
width: 100%;
`;
export const App = () => {

  //Hook para cambiar estado de submit
  const [tareas, cambiarTareas] = useState([{
    id: 1,
    descripcion: "Fabian es lindo",
    completado: false
  }]);

  return (
    <Contenedor__app>
      <Header />
      <FormularioTareas tareas={tareas} cambiarTareas={cambiarTareas} />
    </Contenedor__app>
  )
};
