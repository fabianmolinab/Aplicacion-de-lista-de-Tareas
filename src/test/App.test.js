import React from 'react'

import { shallow } from 'enzyme'
import { App } from '../App'

describe('Pruebas en <App/>', () => {
  test('Debe mostrar correctamente', () => {
    const wrapper = shallow(<App />)
    expect(wrapper).toMatchSnapshot()
  })
})
