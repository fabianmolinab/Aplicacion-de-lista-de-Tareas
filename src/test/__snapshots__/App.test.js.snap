// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en <App/> Debe mostrar correctamente 1`] = `
<Component
  theme={
    Object {
      "background": "#363537",
      "body": "#FFF",
      "text": "#363537",
      "toggleBorder": "#FFF",
    }
  }
>
  <Memo(l) />
  <Switch
    setTheme={[Function]}
    theme="light"
  />
  <styled.div>
    <Header
      cambiarMostrarCompletadas={[Function]}
      mostrarCompletadas={true}
    />
    <FormularioTareas
      cambiarTareas={[Function]}
      tareas={Array []}
    />
    <ListaTareas
      cambiarTareas={[Function]}
      mostrarCompletadas={true}
      tareas={Array []}
    />
  </styled.div>
</Component>
`;
